### ISSUE TITLE
Rename "themes" to "device color"

### ISSUE THREAD

**Issue by sirjamesgray** [retromp3/retro#32]
Since that's how it actually was when buying an iPod the Apple Store 

Themes betrays that this is an emulator artificially showing the color, device color makes the emulator feel psychologically life-like 

Very small detail, I know 🙈

**Comment by sakofchit** [comment]
That is true, but I was planning on adding support for some of the other generations as well (i.e. ipod video, mini, the 3rd gen, etc.)

imo device color would make sense if we were sticking to just 1 generation

**Comment by sirjamesgray** [comment]
Perhaps the it'd be "device" and then "device color" 

But yeah it could be maintained as a single list it it could be shown as "device_name - color" 

Also maybe you could paywall certain devices to help monetize the app? Maybe if apple doesn't let you (since it's on TestFlight) you could tap a Patreon API and only allow paid patrons to access.

I've got other thoughts re: monetizing but I'll gather them and submit them in a more coherent post

**Comment by sakofchit** [comment]
Ah yeah that would make more sense.

As for monetization, I don't really see a need given that we have full, unlimited access to the Spotify/Apple Music APIs––so there's really no cost for me here.

Although there is a Ko-fi link in the app, which I think will suffice for now :)
